src/Database/ODBC/SQLServer.hs

```diff
@@ -271,6 +272,22 @@ newtype Smalldatetime = Smalldatetime
   { unSmalldatetime :: LocalTime
   } deriving (Eq, Ord, Show, Typeable, Generic, Data, FromValue)
 
+newtype Datetimeoffset = Datetimeoffset
```

Comment:
Probably we should document that `ZonedTime's` timezone's zone name will always be empty string and `timeZoneSummerOnly` will always be `False` ?